scene/3d/sprite_3d.cpp

```diff
@@ -366,6 +366,16 @@ void Sprite3D::_draw() {
 		final_rect.position * pixel_size,
 
 	};
+
+	Vector2 src_tsize = Vector2(texture->get_width(), texture->get_height());
+
+	// Properly setup UVs for impostor textures (AtlasTexture).
+	Ref<AtlasTexture> atlas_tex = texture;
+	if (atlas_tex != nullptr) {
```

Comment:
We don't use `nullptr`:
```
scene/3d/sprite_3d.cpp: In member function 'virtual void Sprite3D::_draw()':
scene/3d/sprite_3d.cpp:374:19: error: 'nullptr' was not declared in this scope
  if (atlas_tex != nullptr) {
                   ^
scene/3d/sprite_3d.cpp: In member function 'virtual void AnimatedSprite3D::_draw()':
scene/3d/sprite_3d.cpp:674:19: error: 'nullptr' was not declared in this scope
  if (atlas_tex != nullptr) {
                   ^
```

Reply:
Could you also edit the commit log to something more meaningful? The current log only makes sense if you follow the whole history back to @reduz's comment on GitHub, but doesn't say anything about what this commit actually does (fixing AtlasTexture usage in Sprite3D).